Cargo.toml

```diff
@@ -12,11 +12,17 @@ repository = "https://github.com/m4b/capstone-sys"
 libc = "0.2"
 
 [build-dependencies]
+bindgen = "0.26.3"
 pkg-config = "0.3"
 gcc = "0.3"
 cmake = {optional = true, version = "0.1.2"}
+lazy_static = "0.2"
 
 [features]
+# use bundled capstone and bindings by default
 default = []
```

Comment:
Leaving the comment was a mistake; my intention was to require exactly one of the `use_bundled_capstone` or `use_system_capstone` features.